### PR TITLE
Created Github Workflow for Accessibility Testing (Using Axe)

### PR DESCRIPTION
**Pull request by nwanduka**
Created axe.yml file in the workflows folder that will check for accessibility issues when a push or PR is made

With respect to [this discussion](https://github.com/orgs/camicroscope/discussions/836), I created this workflow to automatically run accessibility checks using the Axe accessibility tool

### CODE REVIEW

File: .github/workflows/axe-accessibility-check.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: axe
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```
Comment (birm) [inline .github/workflows/axe-accessibility-check.yml:6]: Let's include develop too!!

Comment (nwanduka) [reply .github/workflows/axe-accessibility-check.yml]: Done

File: .github/workflows/axe-accessibility-check.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - run: npm run build --if-present
+      - run: npm start & npx wait-on http://localhost:4010
```
Comment (birm) [inline .github/workflows/axe-accessibility-check.yml:26]: We don't have a start script! https://github.com/camicroscope/caMicroscope/blob/dae72cfaaaccf40b82f1f75f16a2a80356a46722/package.json#L6

Comment (nwanduka) [reply .github/workflows/axe-accessibility-check.yml:26]: It appears so. To add a start script, I need to know the entry point to the application. From my research, it is typically specified in the main field of the package.json file as "index.js" or "app.js" or "server.js". But I don't see it. I've scoured the entire repo and I can't find anything remotely close to it.

I was hoping you could point me in the right direction, please.

**Comment by nwanduka** [comment]
I created another PR (#957) to address this. I'll be closing this PR once I get feedback on the other one and a go-ahead from our mentor to close this one.

**Pull request closed without merging**